fbchat/models.py

```diff
@@ -4,471 +4,169 @@
 import enum
 
```

Comment:
Currently, the models are still collected in a single file, but perhaps it would make more sense to spread them out into the other files, like `Message` would be in `send.py`?